cap-ci/cap-nightly-upgrades.yaml

```diff
@@ -0,0 +1,67 @@
+workertags:
+  # caasp4: suse-internal
```

Comment:
Is this commented line on purpose?

Reply:
yes. It was introduced via PR for some reason. Its relevant to access ECP and other internal resources, I think. I kept it as it is part of other CIs as well https://github.com/SUSE/cap/blob/master/cap-ci/cap-pre-release.yaml#L2